### ISSUE TITLE
Missing `role="presentation"` in grid

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3536]
This used to work. It might have regressed with 2017.2. This causes some browsers to announce table rows and columns.
